python/lsst/ts/mtdomecom/constants.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_mtdomecom.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import math
+
+from .power_management import APS_POWER_DRAW, LOUVERS_POWER_DRAW, LWS_POWER_DRAW
+
+# General constants.
+DOME_AZIMUTH_OFFSET = 32.0
+DOME_VOLTAGE = 220.0
+
+# AMCS constants.
+AMCS_NUM_MOTORS = 5
+AMCS_NUM_MOTOR_TEMPERATURES = 13
+AMCS_NUM_ENCODERS = 5
+AMCS_NUM_RESOLVERS = 3
+# Current consumption per motor when moving [A], assuming no acceleration and
+# no wind gust, which is good enough for this simulator, since it ignores both.
+AMCS_CURRENT_PER_MOTOR_MOVING = 40.0
+# Current consumption per motor when crawling [A].
+AMCS_CURRENT_PER_MOTOR_CRAWLING = 4.1
+AMCS_PARK_POSITION = 0.0
+
+# APSCS constants.
+APSCS_NUM_SHUTTERS = 2
+# The number of motors per shutter.
+APSCS_NUM_MOTORS_PER_SHUTTER = 2
+# The shutter is 0% open.
+APSCS_CLOSED_POSITION = 0.0
+# The shutter is 100% open.
+APSCS_OPEN_POSITION = 100.0
+# The shutter speed (%/s). This is an assumed value such that the shutter opens
+# or closes in 10 seconds.
+APSCS_SHUTTER_SPEED = 10.0
+# The motors jitter a bit and this defines the jitter range.
+APSCS_POSITION_JITTER = 2.5e-7
+# Current per motor drawn by the Aperture Shutter [A].
+APSCS_CURRENT_PER_MOTOR = (
+    APS_POWER_DRAW / APSCS_NUM_SHUTTERS / APSCS_NUM_MOTORS_PER_SHUTTER / DOME_VOLTAGE
+)
+
+# CBCS constants.
+CBCS_NUM_CAPACITOR_BANKS = 2
+
+# LCS constants.
+LCS_NUM_LOUVERS = 34
+LCS_NUM_MOTORS_PER_LOUVER = 2
+# Current drawn per louver [A].
+_CURRENT_PER_LOUVER = LOUVERS_POWER_DRAW / LCS_NUM_LOUVERS / DOME_VOLTAGE
```

Comment:
`_CURRENT_PER_LOUVER` --> `CURRENT_PER_LOUVER`

Reply:
This is not a constant that needs to be exported so I will not modify the name. This private constant is used in the definition of the `LCS_CURRENT_PER_MOTOR` constant.